core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -244,7 +249,8 @@ class KafkaApis(val requestChannel: RequestChannel,
               s"on partition $topicPartition failed due to ${Errors.forCode(errorCode).exceptionName}")
```

Comment:
Also, the previous code did not include non existing topics from the server log. Not sure if that was intentional. @hachikuji, do you know?